content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -1917,28 +1917,121 @@ The `filter` element can have the following attributes:
 
          <filter string="Creation Date" name="filter_create_date" date="create_date"/>
 
+   By default, these filters contain a dropdown with different sub-filters that allow you to filter based on months, quarters and years.
+   Additionally, you can create custom sub-filters that allow filtering using domains.
+   These custom filters must have the following attributes: `name`, `string` and `domain`.
+
+   .. example::
+      .. code-block:: xml
+
+         <filter string="Creation Date" name="filter_create_date" date="create_date">
+            <filter name="create_date_last_30_days" string="Last 30 Days" domain="[('create_date', '&gt;', datetime.datetime.combine(context_today() - relativedelta(days=30), datetime.time(23, 59, 59)).to_utc())]"/>
+         </filter>
+
+   Note that all custom filters defined this way are mutually exclusive with each other and with the other sub-filters.
+
    :requirement: Optional
    :type: str
    :default: `''`
 
+.. attribute:: start_month
+   :noindex:
+
+   The earliest month that will show up in the dropdown of a date filter, as an offset relative to the current month.
+   For instance, if the current month is February, a `start_month` with a value of -3 means that the earliest month
+   in the dropdown will be November.
+
+   .. example::
+      .. code-block:: xml
+
+         <filter string="Creation Date" name="filter_create_date" date="create_date" start_month="-3"/>
```

Comment:
IMHO, your textual examples could be moved into the `.. example` sections, to keep the attribute definition short and simple.

```suggestion
   .. example::
      .. code-block:: xml

         <filter string="Creation Date" name="filter_create_date" date="create_date" start_month="-3"/>
         
       If the current month is February, the earliest month shown in the dropdown will be November.
```